account_credit_control/models/credit_control_policy.py

```diff
@@ -201,6 +201,35 @@ def check_policy_against_account(self, account):
             )
         return True
 
+    @api.multi
+    def generate_credit_lines(self, run):
```

Comment:
I would pass the controlling date, so the policy object does not need to know what a run is.

Reply:
We'll see in another PR how to optimize/handle the link between credit control lines and runs.

Reply:
I think passing the controlling data and a dictionary of default values for new lines should be a good approach. So you can pass `{'run_id': ...}`.